var/spack/repos/builtin/packages/vecmem/package.py

```diff
@@ -71,6 +76,7 @@ def cmake_args(self):
             self.define_from_variant("VECMEM_BUILD_SYCL_LIBRARY", "sycl"),
             self.define("BUILD_TESTING", self.run_tests),
             self.define("VECMEM_BUILD_TESTING", self.run_tests),
+            self.define("VECMEM_USE_SYSTEM_LIBS", True),
```

Comment:
Good idea!

(Informational) Flag was added in v0.19.0 (https://github.com/acts-project/vecmem/commit/0641b4810a2762a031d8dce1c4d11a00bd8c7aa4), at which point only google test and benchmark were affected by this. I think it's fine to just apply this flag even for older versions.

Reply:
Yes! I think for vecmem it doesn't _really_ matter because there are few dependencies, but in general Spack should be taking care of this and not CMake, so I'd like for the build system to throw any errors if dependencies are added in the future. :slightly_smiling_face: